cmd/skopeo/copy.go

```diff
@@ -184,7 +187,8 @@ func (opts *copyOptions) run(args []string, stdout io.Writer) error {
 	}
 
 	return retry.RetryIfNecessary(ctx, func() error {
-		_, err = copy.Image(ctx, policyContext, destRef, srcRef, &copy.Options{
+		var manifestBytes []byte
+		manifestBytes, err = copy.Image(ctx, policyContext, destRef, srcRef, &copy.Options{
```

Comment:
```suggestion
		manifestBytes, err := copy.Image(ctx, policyContext, destRef, srcRef, &copy.Options{
```
`err` is not used again outside of this function anyway.